iceoryx_utils/test/moduletests/test_cxx_expected.cpp

```diff
@@ -58,7 +58,7 @@ TEST_F(expected_test, CreateWithValue)
 {
```

Comment:
> Maybe I have forgotten to write tests for it or maybe it was introduced later by someone else?!

I can see that the behavior of `operator bool` has changed in `expected.inl`. However, in `test_cxx_expected.cpp` basically only `get_value()` was replaced by `value()`.  That looks like a test for `operator bool` is missing?!